granary/bluesky.py

```diff
@@ -2247,24 +2259,30 @@ def preview_delete(self, at_uri):
   def base_object(self, obj):
     return base_object(obj)
 
-  def upload_media(self, media):
+  def upload_images(self, images):
     blobs = {}
+    aspects = {}
 
-    for obj in media:
+    for obj in images:
       url = util.get_url(obj, key='stream') or util.get_url(obj)
       if not url or url in blobs:
         continue
 
       with util.requests_get(url, stream=True) as fetch:
         fetch.raise_for_status()
+        # 1,000,000 bytes is the upper limit for a single image on Bluesky
+        image_data = BytesIO(util.FileLimiter(fetch.raw, 1_000_000).read())
```

Comment:
`FileLimiter`, nice! also, maybe read the limit from the lexicon? example: https://github.com/snarfed/granary/blob/c29dd723f8b82ca5874f2122de64b2e9cd2c1fd7/granary/bluesky.py#L2271-L2276

Reply:
Done!